src/views/penaltydetails.njk

```diff
@@ -0,0 +1,100 @@
+{% from "govuk/components/back-link/macro.njk"  import govukBackLink %}
+{% from "govuk/components/input/macro.njk"      import govukInput %}
+{% from "govuk/components/details/macro.njk"    import govukDetails %}
+{% from "govuk/components/button/macro.njk"     import govukButton %}
+
+
+{% block pageTitle %}
+  Tell us the penalty details
+{% endblock %}
+
+{% block beforeContent %}
+  {{ 
+    govukPhaseBanner({
+      tag: {
+        text: "alpha"
+      },
+      html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+    }) 
+  }}
+  {{ 
+    govukBackLink({
+      text: "Back",
+      href: "signin"
+    }) 
+  }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        {{ govukErrorSummary
+          ({
+            titleText: "There is a problem with the details you gave us",
+            errorList: validationResult.errors
+          }) if validationResult and validationResult.errors.length > 0 
+        }}
+      <h1 class="govuk-heading-xl">
+        What are the penalty details?
+      </h1>
+      <form method="post" action="">
+        {{ 
+          govukInput({
+            label: {
+              text: "Company number"
+            },
+            hint: {
+              text: "This is the 8-character reference issued by Companies House when the company was set up."
+            },
+            errorMessage: validationResult.getErrorForField('companyNumber') if validationResult,
+            classes: "govuk-input--company-number",
+            id: "company-number",
+            name: "companyNumber",
+            value: companyNumber
+          }) 
+        }}
+        {{ 
+          govukInput({
+            label: {
+              text: "Reference number"
+            },
+            hint: {
+              text: "This is the 9-character reference shown on the penalty notice."
+            },
+            errorMessage: validationResult.getErrorForField('penaltyReference') if validationResult,
+            classes: "govuk-input--penalty-reference",
+            id: "penalty-reference",
+            name: "penaltyReference",
+            value: penaltyReference
+          }) 
+        }}
+        <div class="govuk-form-group">
+          {% set detailsHTML %}
+          <p>
+            Your company number and reference number are shown at the bottom of your penalty notice.
+          </p>
+          <img src='/public/images/penalty-notice-image.png' class='call-out-image' alt=''/>
+          {% endset -%}
+          {{ 
+            govukDetails({
+              summaryText: "Help with company number and penalty reference number",
+              html: detailsHTML
+            }) 
+          }}
+        </div>
+        {{ 
+          govukButton({
+            text: "Continue",
+            attributes: {
+              "data-event-id": "continue-button",
```

Comment:
What is that `data-event-id` for?

Reply:
I added this as i thought it was required for the id on the continue button. I'll remove this for now